BookingsApi/BookingsApi.DAL/Migrations/20220128102816_AddNewCaseHearingTypesAndRolesToFamilyType.cs

```diff
@@ -0,0 +1,116 @@
+                    {809, HearingRoles.MacKenzieFriend, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 262},
+                    {810, HearingRoles.Representative, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 262},
+                    {811, HearingRoles.Solicitor, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 262},
+                    {812, HearingRoles.Witness, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 262},
+                    // Respondent(263)
+                    {813, HearingRoles.Barrister, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 263},
+                    {814, HearingRoles.Expert, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 263},
+                    {815, HearingRoles.Intermediary, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 263},
+                    {816, HearingRoles.Interpreter, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 263},
+                    {817, HearingRoles.LitigantInPerson, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 263},
+                    {818, HearingRoles.LitigationFriend, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 263},
+                    {819, HearingRoles.MacKenzieFriend, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 263},
+                    {820, HearingRoles.Representative, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 263},
+                    {821, HearingRoles.Solicitor, UserRoleForHearingRole.UserRoleId[UserRoles.Representative], 263},
+                    {822, HearingRoles.Witness, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 263},
+                    // Panel Member(264)
+                    {823, HearingRoles.PanelMember, UserRoleForHearingRole.UserRoleId[UserRoles.PanelMember], 264},
+                    // Observer(265)
+                    {824, HearingRoles.Observer, UserRoleForHearingRole.UserRoleId[UserRoles.Individual], 265},
+                    //Judge(266)
+                    {825, HearingRoles.Judge, UserRoleForHearingRole.UserRoleId[UserRoles.Judge], 266},
+                });
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DeleteData(nameof(CaseType), "Id", 41);
+            
+            for (int hearingTypeId = 205; hearingTypeId <= 213; hearingTypeId++)
+            {
+                migrationBuilder.DeleteData(nameof(HearingType), "Id", hearingTypeId);
+            }
+            
+            for (int caseRoleId = 261; caseRoleId <= 266; caseRoleId++)
+            {
+                migrationBuilder.DeleteData(nameof(CaseRole), "Id", caseRoleId);
+            }
+            
+            for (int hearingRoleId = 793; hearingRoleId <= 825; hearingRoleId++)
+            {
+                migrationBuilder.DeleteData(nameof(HearingRole), "Id", hearingRoleId);
+            }
```

Comment:
Are these ok being deleted in the same order as they are inserted? Could dependencies cause issues or does it only matter that they are all gone by the end

Reply:
I see other migrations that have this order so happy enough to approve and you can resolve this comment but just to confirm.